tests/ui/feature-gates/feature-gate-precise_pointer_size_matching.stderr

```diff
@@ -6,7 +6,6 @@ LL |     match 0usize {
    |
    = note: the matched value is of type `usize`
    = note: `usize` does not have a fixed maximum value, so half-open ranges are necessary to match exhaustively
-   = help: add `#![feature(precise_pointer_size_matching)]` to the crate attributes to enable precise `usize` matching
```

Comment:
Could we add a suggestion to use half-open range patterns so that users of this feature gate (if there are any) are directed towards code that will work for them?

Reply:
I was thinking the same, may I merge as-is?

Reply:
Oh, yeah, didn't notice that - apologies.